cmd/daemon/main.go

```diff
@@ -383,6 +383,7 @@ type Config struct {
 	ZeroconfEnabled       bool      `koanf:"zeroconf_enabled"`
 	ZeroconfPort          int       `koanf:"zeroconf_port"`
 	DisableAutoplay       bool      `koanf:"disable_autoplay"`
+	InterfacesToAdvertise []string  `koanf:"interfaces_to_advertise"`
```

Comment:
I think this should mention Zeroconf in the name, it's a bit vague as is.

Reply:
looks like resonable changes